External/Plugins/ResultsPanel/Helpers/ResultsPanelHelper.cs

```diff
@@ -1,83 +1,249 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
 using PluginCore;
 using PluginCore.Localization;
 using PluginCore.Managers;
 using WeifenLuo.WinFormsUI.Docking;
 
 namespace ResultsPanel.Helpers
 {
-    class ResultsPanelHelper
+    internal static class ResultsPanelHelper
```

Comment:
Static class is much better in the cases of:

- When the class only needs one instance
- The class needs to be initialised as soon as the functionality is needed
- The class lasts for the entire duration of the assembly's life
- The class doesn't need derived types or interface

If a singleton was to be used in this case, it's actually worse, because:

- It requires the class to have a constructor
- There is a cost of class instantiation
- There is the class definition, and also the instance object definition lying in the memory
- The object lies in the part of the memory which is targeted by the GC, meaning although negligible, it's a permanent object never to be GC'ed until the assembly unloads, only hindering the GC process by having to be moved around.